src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -355,6 +357,9 @@ class primitive_inst {
     // List of depandant shape_of primitives for shape_of subgraphs
     std::vector<primitive_inst*> dependant_shape_of_insts;
 
+    // List of dependant primitives for state initializer subgraphs
+    std::vector<primitive_inst*> state_initializers;
```

Comment:
That can also be moved to cldnn::network member. It's not a common property of primitive_inst, so it shouldn't be a part of base class.

Reply:
moved from `primitive_inst` to `network`